### PR TITLE
edit songs impl

### PR DESCRIPTION
**Pull request by Aravind-Siddharth-Sakthivel**

### CODE REVIEW

File: models/songs.model.js [hunk 1]
```diff
@@ -32,7 +34,30 @@ Song.findById = function (id, result) {
...
+        result(null, res);
+      }
+    }
+  );
+};
+
+Song.findAlbums = function (result) {
+  dbConn.query("Select album from songs group by album", function (err, res) {
+    if (err) {
+      result(null, err);
```
Comment (n-grounds) [inline models/songs.model.js:51]: shouldn't the error be passed as the first argument to the `result` function?

Comment (Aravind-Siddharth-Sakthivel) [reply models/songs.model.js:51]: Yes, I agree. I can pass the error more simple as return err;

Comment (Aravind-Siddharth-Sakthivel) [reply models/songs.model.js:51]: we will be adding error handlers & global error handlers for all queries/endpoints in the next push

**Comment by n-grounds** [comment]
Best practice is to have a `dev` branch where each feature branch should be merged into, then [usually once at the end of each Sprint] you can merge `dev` branch into `main`.  That way `main` branch is always your Sprint-by-Sprint ready-to-ship product, and `dev` is the combination of whatever work has been done so far in the current Sprint.

**Pull request merged**